lib/src/component/aria_mixin.dart

```diff
@@ -34,13 +35,16 @@ abstract class $AriaPropsMixin {
   static const PropsMeta meta = _$metaForAriaPropsMixin;
 }
 
+///See: <https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes>
+
```

Comment:
For this - when "documenting" an entire page / library, we usually put them above the library directive. In this case - that is on line 15...

```dart
/// See: <https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes>
library over_react.aria_mixin;
```